css/index.css

```diff
@@ -0,0 +1,476 @@
+*,
```

Comment:
Do some research about the best practices on how to write proper css files.
I dont know much about this particular topic but it will help you to write better CSS, hopefully